src/Game.js

```diff
@@ -48,7 +54,8 @@ export default class Game {
     onMouseOut({ target }) {
         if (!this.validShape(target)) return;
 
-        target.resetColor();
+        target.changeStrokeThickness()
```

Comment:
change to what? Consider a separate function to reset stroke thickness to default

Reply:
By default the thickness parameter here will be DEFAULT_THICKNESS. Its a way of overloading in javascript by setting default parameter values.